js/adapt-articleView.js

```diff
@@ -1,508 +1,497 @@
 define([
-    'core/js/adapt',
-    'core/js/views/articleView'
+  'core/js/adapt',
+  'core/js/views/articleView'
 ], function(Adapt, AdaptArticleView) {
 
-    function debounce(callback, timeout) {
+  function debounce(callback, timeout) {
```

Comment:
seems odd that this has its own version of debounce rather than just using `_.debounce`... not saying it needs to be fixed as part of this, just making a note for reference really!

Reply:
It's a cancellable debounce,

Reply:
According to the underscore docs for `_.debounce`:
> If you need to cancel a scheduled debounce, you can call `.cancel()` on the debounced function.
Or is that not quite the same as this implementation?

Reply:
Ah, looks like the `.cancel` feature was only added in Underscore v1.9.0 (released April 18, 2018) so may not have been available when you originally wrote this code...

Reply:
(and we only updated Adapt from underscore v1.8.3 to v1.9.1 in July/August 2018...)

Reply:
The should be removed in favour of `_.debounce`